cli/Valet/Site.php

```diff
@@ -503,7 +503,7 @@ public function secure(string $url, ?string $siteConf = null, int $certificateEx
     /**
      * Renews all domains with a trusted TLS certificate.
      */
-    public function renew($expireIn): void
+    public function renew($expireIn = 368): void
```

Comment:
Why 368 tho? Or is it supposed to be 365 = number of days in a year?

Reply:
Yes, 365 plus a couple more so that it's not always the same.